src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
+    <Description>OpenTelemetry protocol exporter for OpenTelemetry .NET</Description>
+    <PackageTags>$(PackageTags);OTLP</PackageTags>
+  </PropertyGroup>
```

Comment:
If we keep this as a separate package for now, I have not yet defined a tag for `MinVer`. As it stands, this should result in this package being versioned the same as our instrumentation packages. Is that ok with folks? Or does it make sense to make a new tag just for this package?

Reply:
this should be fine now. Once we are ready, we can do a release of all the non-core packages which will include this one too.